distributions/openhab/merge-addon-info.groovy

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
```

Comment:
Is this license header needed, and if so, is there a configuration in this repository for automatically updating copyright year in license headers?

Reply:
I added it because we also have it on our other [Groovy file](https://github.com/openhab/openhab-core/blob/7bac24e58f057ddb739194a7fe370ab6c42758c0/bundles/org.openhab.core.semantics/model/generateTagClasses.groovy#L1-L12).

There is no license-maven-plugin config currently but we can add it before 2025 when the header survives the merge request.

Reply:
I created #1627 for this